README.md

```diff
@@ -84,136 +85,88 @@ An agent component for Java system.
 
 ## QuickStart
 
-### Download
-Download `easeagent.jar` from releases [releases](https://github.com/megaease/easeagent/releases).
+### Get And Set Environment Variable
 
-or
+download easeagent or build easeagent from source then set $EASE_AGENT_PATH
+#### Download
+```
+$ cd ~/easeagent #[Replace with agent path]
+$ export EASE_AGENT_PATH=`pwd` # export EASE_AGENT_PATH=[Replace with agent path]
+$ mkdir plugins
+```
+
+Download `easeagent.jar` from releases [releases](https://github.com/megaease/easeagent/releases).
 
 ```
 $ curl -Lk https://github.com/megaease/easeagent/releases/download/{tag}/easeagent.jar -O
```

Comment:
the URL can be replaced by lastest version: https://github.com/megaease/easeagent/releases/latest/download/easeagent.jar